.github/workflows/spec_test_on_nuttx.yml

```diff
@@ -37,32 +37,32 @@ jobs:
           # {
           #   config: "boards/arm64/qemu/qemu-armv8a/configs/nsh",
           #   target: "aarch64_vfp",
-          #   use_fpu: true
+          #   fpu_type: "fp"
           # },
           # {
           #   config: "boards/arm/imx6/sabre-6quad/configs/nsh",
           #   target: "thumbv7",
-          #   use_fpu: false
+          #   fpu_type: "none"
           # },
           {
             config: "boards/arm/imx6/sabre-6quad/configs/nsh",
             target: "thumbv7_vfp",
-            use_fpu: true
+            fpu_type: "dp"
```

Comment:
Seems that "dp" isn't handled in below?

Reply:
Yes, no need to handle it since it's enabled by default if DPFPU is available